scripts/release/README.md

```diff
@@ -2,26 +2,31 @@
 
 #### Web Widgets
 1. Create a release in Github
-1. Add a tag formatted like ${PackageName}-v${Major}.${Minor}.${Patch}
+1. Add a tag to the commit you want to create a release from. The tag should be formatted like ${PackageName}-v${Major}.${Minor}.${Patch}
     - Example: `maps-web-v2.0.2`
-1. That's it! The content should now be uploaded to the marketplace. Double check to verify.
+1. That's it! 
+    - The content should now be released in Github.
+    - The content should now be released in the MX Marketplace. Double check to verify.
 
 
 #### Web Modules
-1. Create a release in Github
-1. Add a tag formatted like ${PackageName}-v${Major}.${Minor}.${Patch}
+1. Add a tag to the commit you want to create a release from. The tag should be formatted like ${PackageName}-v${Major}.${Minor}.${Patch}
     - Example: `nanoflow-actions-web-v2.0.0`
-1. That's it! The content should now be uploaded to the marketplace. Double check to verify.
+1. That's it! 
+    - The content should now be released in Github.
+    - The content should now be released in the MX Marketplace. Double check to verify.
 
 #### Native Modules
-1. Create a release in Github
-1. Add a tag formatted like ${PackageName}-v${Major}.${Minor}.${Patch}
+1. Add a tag to the commit you want to create a release from. The tag should be formatted like ${PackageName}-v${Major}.${Minor}.${Patch}
```

Comment:
If the steps to release native and hybrid modules are the same, could you squash the two sections into one?

Reply:
Yes we can, but I think it's easier to understand if we leave it separate :)